drizzlepac/processInput.py

```diff
@@ -323,8 +323,6 @@ def reportResourceUsage(imageObjectList, outwcs, num_cores,
 
 def getMdriztabPars(input):
     """ High-level function for getting the parameters from MDRIZTAB
-
-    Used primarily for TEAL interface.
```

Comment:
Do not remove comment.  Just add text the function may be deprecated as the use of the TEAL GUI is being disabled.